src/components/FeatureWalkthrough.tsx

```diff
@@ -0,0 +1,128 @@
+import { FunctionComponent, useEffect } from 'react'
+
+import classNames from 'classnames'
+import { startCase } from 'lodash'
+import Link from 'next/link'
+
+import { ContentSection } from '@components'
+import { buttonStyle, buttonLocation } from '@data'
+import { Features, VideoElement } from '@interfaces/featureWalkthrough'
+
+import { COLORS } from './ContentSection'
+
+interface Props {
+    features: Features[]
+    className?: string
+    color?: keyof typeof COLORS
+}
+
+export const FeatureWalkthrough: FunctionComponent<Props> = ({ features, className = '', color = 'none' }) => {
+    useEffect(() => {
+        const videos = features.map(
+            (vid, index): VideoElement => ({
+                el: document.querySelector(`.video-${index}`),
+                paused: true,
+            })
+        )
+
+        if (window.IntersectionObserver) {
+            for (const vid of videos) {
+                const observer = new IntersectionObserver(
+                    entries => {
+                        const currentVideo = entries[0]
+
+                        if (!currentVideo.isIntersecting && !vid.paused) {
+                            vid?.el?.pause()
+                            vid.paused = true
+                        } else if (vid.paused) {
+                            vid?.el?.play()
+                            vid.paused = false
+                        }
+                    },
+                    { threshold: 1 }
+                )
+
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                observer.observe(vid.el!)
+            }
+        }
+    }, [features])
+
+    return (
+        <ContentSection color={`${color}`} className={`${className}`}>
```

Comment:
Since `color` and `className` don't have any other strings getting assigned to their attribute, I don't think we need to interpolate here

```suggestion
52:         <ContentSection color={color} className={className}>